tests/lora/test_unexpected_modules.py

```diff
@@ -0,0 +1,50 @@
+from typing import List
+
+import pytest
+import torch
+
```

Comment:
Can we merge this scirpt with `test_lora_checkpoints.py`, it seems that they are similar